internal/testdata/gen/gen.go

```diff
@@ -43,12 +41,29 @@ const (
 )
 
 var (
-	outputPath = flag.String("output_path", "./internal/testdata/", "Output path for private keys and certificates")
+	outputPath      = flag.String("output_path", "./internal/testdata/", "Output path for private keys and certificates")
+	notBeforeString = flag.String("not_before", "2024-12-05T18:05:50.000Z", "Start of the range of certs to be generated. RFC3339 UTC format, e.g: 2024-01-02T15:04:05Z.")
+)
+
+var (
+	// From RFC6962 Section 3.1. To identify pre-certs.
+	cTPrecertPoisonOID = asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 11129, 2, 4, 3}
```

Comment:
nit: `ctPrecert...` to match the `ctPoison` case-style below

Reply:
good catch, thanks!